src/fontra/views/editor/edit-tools-pointer.js

```diff
@@ -124,15 +170,28 @@ export class PointerTool extends BaseTool {
     }
 
     sceneController.hoveredGlyph = undefined;
+    const resizeHandle = this.getResizeHandle(initialEvent, initialSelection);
 
-    if (initiateRectSelect) {
+    if (initiateRectSelect && !resizeHandle) {
       return await this.handleRectSelect(eventStream, initialEvent, initialSelection);
-    } else if (initiateDrag) {
+    } else if (initiateDrag && !resizeHandle) {
       this.sceneController.sceneModel.initialClickedPointIndex =
         initialClickedPointIndex;
       const result = await this.handleDragSelection(eventStream, initialEvent);
       delete this.sceneController.sceneModel.initialClickedPointIndex;
       return result;
+    } else if (
+      resizeHandle &&
+      this.editor.visualizationLayersSettings.model["fontra.resize.selection"]
+    ) {
+      sceneController.selection = initialSelection;
+      this.sceneController.sceneModel.clickedResizeHandle = resizeHandle;
+      await this.handleDragSelectionBoundsResize(
```

Comment:
Please call this method `handleBoundsResize`. We're not dragging the selection, were resizing by bounds.

Reply:
Thanks. Done.